x-pack/test/functional/apps/dashboard/group3/reporting/download_csv.ts

```diff
@@ -81,7 +81,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       }
     });
 
-    describe('Default Saved Search Data', () => {
```

Comment:
Let's reset the functional tests back to the version from main with:
```
git checkout elastic/main x-pack/test
```

Since all the types are working and available, the tests should stay as-is.